projects/superstate/index.js

```diff
@@ -1,7 +1,14 @@
+const { get } = require("../helper/http");
+
 module.exports = {
     methodology: "Sums the total supplies of Superstate's issued tokens.",
 };
 
+async function USTBPrice() {
```

Comment:
is there some way to pull this price on chain? like via chainlink or pyth

Reply:
no because it is a RWA.  our api relies on a third party to calculate our NAV

Reply:
but USTB is traded in stock market right? did you check if pyth has support for that? issue with relying on your api is, api might rug for whatever reason and we will have  a broken adapter in our hands

Reply:
It is a private fund and is not tradable by the public (token contract has an allowlist).  The value of the fund is determined by NAV Consulting (https://www.navconsulting.net) which we proxy.

You'll be able to pull from Coingecko soon, but they're also just pulling their data from this endpoint.

Reply:
When we do have an onchain oracle, it'll read from our API anyway.